p/paping/paping_rhel_8.4.sh

```diff
@@ -0,0 +1,26 @@
+# -----------------------------------------------------------------------------
+#
+# Package       : paping
+# Version       : v1.5.5
+# Source repo   : https://github.com/koolhazz/paping.git
+# Tested on     : RHEL 8.4
+# Script License: Apache License Version 2.0
+# Maintainer    : sachin.kakatkar@ibm.com
```

Comment:
Please add `Language : ` field in the script header section.

You can find details in recent PRs.

Reply:
Added Language section.